src/main/java/org/openstreetmap/atlas/geography/atlas/command/OsmPbfToAtlasSubCommand.java

```diff
@@ -0,0 +1,207 @@
+            Optionality.OPTIONAL);
+    private static final Switch<File> COUNTRY_MAP_PARAMETER = new Switch<>("country-boundary-map",
+            "Path to a WKT or shp file containing a country boundary map", File::new,
+            Optionality.OPTIONAL);
+    private static final Switch<Boolean> COUNTRY_SLICING_PARAMETER = new Switch<>("country-slicing",
+            "Whether to perform country slicing (boolean)", Boolean::new, Optionality.OPTIONAL);
+
+    // Way Sectioning Parameter
+    private static final Switch<Boolean> WAY_SECTION_PARAMETER = new Switch<>("way-section",
+            "Whether to perform way sectioning (boolean)", Boolean::new, Optionality.OPTIONAL);
+
+    @Override
+    public String getDescription()
+    {
+        return DESCRIPTION;
+    }
+
+    @Override
+    public String getName()
+    {
+        return NAME;
+    }
+
+    @Override
+    public SwitchList switches()
+    {
+        return new SwitchList().with(INPUT_PARAMETER, OUTPUT_PARAMETER, EDGE_FILTER_PARAMETER,
+                NODE_FILTER_PARAMETER, RELATION_FILTER_PARAMETER, WAY_FILTER_PARAMETER,
+                WAY_SECTION_FILTER_PARAMETER, LOAD_RELATIONS_PARAMETER, LOAD_WAYS_PARAMETER,
+                COUNTRY_CODES_PARAMETER, COUNTRY_MAP_PARAMETER, COUNTRY_SLICING_PARAMETER,
+                WAY_SECTION_PARAMETER);
+    }
+
+    @Override
+    public void usage(final PrintStream writer)
+    {
+        writer.println("-pbf=/path/to/pbf : pbf to convert");
+        writer.println("-output=/path/to/output/atlas : Atlas file to output to");
+        writer.println("-edge-filter=/path/to/json/edge/filter : json filter to determine Edges");
+        writer.println("-node-filter=/path/to/json/node/filter : json filter for OSM nodes");
+        writer.println(
+                "-relation-filter=/path/to/json/relation/filter : json filter for OSM relations");
+        writer.println("-way-filter=/path/to/json/way/filter : json filter for OSM ways");
+        writer.println("-load-relations=boolean : whether to load Relations; defaults to true");
+        writer.println("-load-ways=boolean : whether to load ways; defaults to true");
+        writer.println(
+                "-country-codes=list,of,ISO3,codes : countries from the country map to convert");
+        writer.println(
+                "-country-boundary-map=/path/to/WKT/or/shp : a WKT or shp file containing a country boundary map");
+        writer.println(
+                "-country-slicing=boolean : whether to perform country slicing; defaults to true");
+        writer.println(
+                "-way-sectioning=boolean : whether to perform way sectioning; defaults to true");
+        writer.println(
+                "-way-section-filter=/path/to/json/way/section/filter : json filter to determine where to way section");
+    }
+
+    @Override
+    public int execute(final CommandMap map)
+    {
+        new OsmPbfLoader((File) map.get(INPUT_PARAMETER), this.getAtlasLoadingOption(map))
```

Comment:
Here, I think we should use the latest ingest, that goes through raw Atlas. I think this is the way (@MikeGost can correct me):
```java
File pbf, output;
AtlasLoadingOption option;

Atlas atlas = new RawAtlasGenerator(pbf, option).build();
if (countrySlicing)
{
    atlas = new RawAtlasCountrySlicer(countryList, countryBoundaries).slice(atlas);
}
atlas = new WaySectionProcessor(atlas, option).run();
atlas.save(output);
```
@MikeGost Maybe we should have an easy one-liner available that does all that somewhere, when the pbf->atlas loading use case is simple like that.

Reply:
Agreed, this should flow through the raw atlas flow instead of the legacy atlas creation mechanism. This is also strikingly similar to the work that @lucaspcram did here: https://github.com/osmlab/atlas-generator/pull/53. We may want to consolidate the tools into a single one.

Reply:
See `RawAtlasIntegrationTest` for some examples of the flow.

Reply:
@matthieun - I have a backlog item to refactor AtlasLoadingOption that would automatically allow you to build complete atlases, without having to perform each step (create, slice, section) yourself. Definitely valuable functionality that can be reused here.

Reply:
It looks easy enough to change this over to the raw atlas flow. The bigger questions seems to be if we want to consolidate commands. I did not realize the atlas generator command existed, and I agree it has very similar functionality. The one big difference I see is that the atlas generator command is built around the sharded pbf/atlas workflow, whereas this command is made to be a more general flow.